src/kitconcept/seo/events.py

```diff
@@ -0,0 +1,13 @@
+from plone.app.contenttypes.interfaces import IImage
+from zope.component import adapter
+from ZPublisher.interfaces import IPubAfterTraversal
+from ZPublisher.pubevents import PubAfterTraversal
+
+
+@adapter(IPubAfterTraversal)
+def after_image_pub_traversal(event: PubAfterTraversal):
+    """Record the view as the HTTP route after traversal"""
+    request = event.request
+    context = event.request["PARENTS"][0]
+    if IImage.providedBy(context) and context.seo_noindex:
```

Comment:
This needs to handle objects that provide IImage but don't have the SEO behavior enabled.

I'd also like a test for this feature.